contracts/DCAStrategies/DCAStrategies/DCAStrategiesManagementHandler.sol

```diff
@@ -9,45 +9,56 @@ abstract contract DCAStrategiesManagementHandler is IDCAStrategiesManagementHand
     uint96 latestVersion;
   }
 
-  mapping(uint80 => StrategyOwnerAndVersion) internal _strategies;
-  mapping(string => uint80) public getStrategyIdByName;
-  mapping(uint80 => string) public getStrategyNameById;
-  mapping(bytes32 => IDCAStrategies.ShareOfToken) internal _tokenShares;
+  /// @inheritdoc IDCAStrategiesManagementHandler
+  mapping(string => uint80) public strategyIdByName;
+  /// @inheritdoc IDCAStrategiesManagementHandler
   uint80 public strategyCounter;
+  mapping(uint80 => StrategyOwnerAndVersion) internal _strategies;
+  mapping(uint80 => string) internal _strategyNameById;
+  mapping(bytes32 => IDCAStrategies.ShareOfToken[]) internal _tokenShares;
 
-  function _checkTokenSharesSanity(IDCAStrategies.ShareOfToken memory _tokens) internal pure returns (bool) {
+  function _checkTokenSharesSanity(IDCAStrategies.ShareOfToken[] memory _tokens) internal pure returns (bool) {
```

Comment:
Could you move the internal functions to the bottom of the file, after the public/external functions?